src/Rings/solving.jl

```diff
@@ -0,0 +1,54 @@
+export real_solutions
+
+@doc Markdown.doc"""
+    real_solutions(I::MPolyIdeal, <keyword arguments>)
+
+Given an ideal `I` with a finite solution set over the complex numbers, return a pair `r,p` where `p` is the rational parametrization of the solution set and `r` represents the real roots of `Ì`  with a given precision (default 32 bits).
+See [BES21](@cite) for more information.
+
+**Note**: At the moment only QQ is supported as ground field. If the dimension of `I`
+is greater then zero an empty array is returned.
+
+# Arguments
+- `Ì::MPolyIdeal`: input ideal.
+- `initial_hts::Int=17`: initial hash table size `log_2`.
+- `nr_thrds::Int=1`: number of threads for parallel linear algebra.
```

Comment:
No, this is calling C code from within Julia. At this point we cannot assume the globally set number of threads in Julia is the one to be chosen here. Assume that an Oscar user allows in Julia 4 threads, running 4 Groebner basis computations in parallel. If msolve assumes that it can also take 4 threads then overall 16 threads would be running in parallel.

Reply:
After some discussion with Claus I think that setting `nr_thrds=1` for now is reasonable.